README.md

```diff
@@ -81,19 +81,19 @@ You can use the Parcel component either by npm installing the library and import
    Note that if the `<Parcel>` component is being rendered by a single-spa application that uses single-spa-react, it is **unnecessary** to pass in the prop, since `<Parcel>` can get the prop
    from [SingleSpaContext](#singlespacontext)
 - `handleError` (optional): A function that will be called with errors thrown by the parcel. If not provided, errors will be thrown on the window, by default.
+- `parcelWasMounted` (optional): A function that will be called when the parcel finishes loading and mounting.
 
 #### Examples
 ```jsx
 import Parcel from 'single-spa-react/parcel'
 import * as parcelConfig from './my-parcel.js'
 
-// config and wrapWith are required. The parcel will be mounted inside of the
+// config is required. The parcel will be mounted inside of the
```

Comment:
wrapWith actually isn't required

Reply:
We'll need to update this on the website as well